### PR TITLE
add packages libtorrent-rasterbar and qbittorrent

### PR DESCRIPTION
**Pull request by starius**
**libtorrent** (aka libtorrent-rasterbar) is a feature complete C++ bittorrent implementation focusing on efficiency and scalability. It runs on embedded devices as well as desktops. It boasts a well documented library interface that is easy to use. It comes with a simple bittorrent client demonstrating the use of the library.

The main goals of libtorrent are:

  *  to be cpu efficient
  *  to be memory efficient
  *  to be very easy to use

http://www.libtorrent.org/

The **qBittorrent** project aims to provide a Free Software alternative to µtorrent. Additionally, qBittorrent runs and provides the same features on all major platforms (Linux, Mac OS X, Windows, OS/2, FreeBSD).

qBittorrent is based on Qt4 toolkit and libtorrent-rasterbar.

http://www.qbittorrent.org/

### CODE REVIEW

File: src/libtorrent-rasterbar-2-geoip-win32.patch [hunk 1]
```diff
@@ -0,0 +1,29 @@
+This file is part of MXE.
+See index.html for further information.
+
+From 258068273e70f25b723c4813805e76f92b18264c Mon Sep 17 00:00:00 2001
+From: Boris Nagaev <bnagaev@gmail.com>
+Date: Sat, 29 Aug 2015 11:03:49 +0200
+Subject: [PATCH] GeoIP.c: disable POSIX section
+
+WIN32 is not defined for some reason.
```
Comment (TimothyGu) [inline src/libtorrent-rasterbar-2-geoip-win32.patch]: You can and should use `_WIN32`.

Comment (starius) [inline src/libtorrent-rasterbar-2-geoip-win32.patch]: Fixed.

**Comment by tonytheodore** [comment]
Following from the discussion in #808, qBittorrent is not in the scope of MXE - could you separate the pull request?

**Comment by starius** [comment]
Replaced with #825

**Pull request closed without merging**